pyvista/plotting/plotting.py

```diff
@@ -1554,6 +1564,27 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
 
         ##### Plot a single PyVista mesh #####
 
+        silhouette_params = dict(rcParams['silhouette'])
+        if isinstance(silhouette, dict):
+            silhouette_params.update(silhouette)
+            silhouette = True
+        if silhouette:
+            if silhouette_params["decimate"] is None:
+                silhouette_mesh = mesh
+            else:
+                silhouette_mesh = mesh.decimate(silhouette_params["decimate"])
+            alg = _vtk.vtkPolyDataSilhouette()
+            alg.SetInputData(silhouette_mesh)
+            alg.SetCamera(self.camera)
+            alg.SetEnableFeatureAngle(silhouette_params["feature_angle"])
+            mapper = make_mapper(_vtk.vtkDataSetMapper)
+            mapper.SetInputConnection(alg.GetOutputPort())
+            _, prop = self.add_actor(mapper)
+            prop.SetColor(parse_color(silhouette_params["color"]))
+            prop.SetOpacity(silhouette_params["opacity"])
+            prop.SetLineWidth(silhouette_params["line_width"])
+            del mapper
```

Comment:
Is it important to remove this name before the method finishes?

Reply:
I removed it :+1: